packages/babel-preset-latest/src/index.js

```diff
@@ -1,11 +1,16 @@
-const { buildPreset } = require("babel-preset-es2015");
+import { buildPreset } from "babel-preset-es2015";
+import * as presetES2016 from "babel-preset-es2016";
+import * as presetES2017 from "babel-preset-es2017";
 
-module.exports = function preset(context, opts = {}) {
+//Rather than exporting a default object to represent the preset, we can
+//also export a default function instead, as this preset demonstrates.
+//This allows one to further configure a preset by way of specific options.
```

Comment:
same here: one space after // please
We should add the linting rule for that :)